lib/mllp/echo_dispatcher.ex

```diff
@@ -1,7 +1,8 @@
-defmodule MLLP.DefaultDispatcher do
+defmodule MLLP.EchoDispatcher do
```

Comment:
By renaming here you will also need to update the docs (receiver.ex line 51) to use the new name.

As is `iex(1)> h MLLP.Receiver.start` tells user to do it wrong

Reply:
Good catch.